tests/auth_tests/test_views.py

```diff
@@ -943,14 +944,27 @@ def test_user_not_change(self):
         self.assertEqual(row.change_message, 'No fields changed.')
 
     def test_user_change_password(self):
+        user_change_url = reverse('auth_test_admin:auth_user_change', args=(self.admin.pk,))
+        password_change_url = reverse('auth_test_admin:auth_user_password_change', args=(self.admin.pk,))
+
+        response = self.client.get(user_change_url)
+        # Test the link inside password field help_text.
+        rel_link = re.search(
+            r'you can change the password using <a href="([^"]*)">this form</a>', response.content
+        ).groups()[0]
+        self.assertEqual(
+            os.path.normpath(user_change_url + rel_link),
```

Comment:
Does Windows use `/` or `\` in normalized paths? In case of `\`, does the test still pass?